index.html

```diff
@@ -4210,8 +4210,8 @@ <h2>
 used to communicate this metadata MUST be a <a data-cite="INFRA#maps">map</a>
 of properties. Each property name MUST be a <a
 data-cite="INFRA#string">string</a>. Each property value MUST be a <a
-data-cite="INFRA#string">string</a>, <a data-cite="INFRA#maps">map</a>, <a
-data-cite="INFRA#lists">list</a>, <a data-cite="INFRA#boolean">boolean</a>, or
+data-cite="INFRA#string">string</a>, <a data-cite="INFRA#maps">map</a>, an <a
+data-cite="INFRA#ordered-set">ordered set</a>, <a data-cite="INFRA#boolean">boolean</a>, or
```

Comment:
Per @msporny 's comment https://github.com/w3c/did-core/pull/522#discussion_r551029641, probably you want to reinstate "list" here as well.

Reply:
Yep. Changed it in https://github.com/w3c/did-core/pull/522/commits/fd01d9c3cda9ec57b78057c97edec0af6cfd992b.